src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
-management.endpoints.web.base-path=/
+management.endpoints.web.base-path=/disqualified-officers-delta-consumer
 management.endpoints.web.path-mapping.health=healthcheck
```

Comment:
I think this will need a / before the healthcheck? e.g. `/healthcheck` 
this would then be the full url `/disqualified-officers-delta-consumer/healthcheck`?

Reply:
It seems to work as it is locally?

```
pd12586:~ vjackson$ curl localhost:8080/disqualified-officers-delta-consumer/healthcheck
{"status":"UP"}
```

Reply:
fair enough :)